src/translation/translation_nb_NO.ts

```diff
@@ -944,42 +944,42 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
         <location filename="../clientdlgbase.ui" line="400"/>
         <location filename="../clientdlgbase.ui" line="438"/>
         <source>ms</source>
-        <translation type="unfinished"></translation>
+        <translation>ms</translation>
     </message>
     <message>
         <location filename="../clientdlgbase.ui" line="498"/>
         <source>&amp;Mute Myself</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Forstum deg selv</translation>
     </message>
     <message>
         <location filename="../clientdlgbase.ui" line="505"/>
         <source>&amp;Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Innstillinger</translation>
     </message>
     <message>
         <location filename="../clientdlgbase.ui" line="512"/>
         <source>&amp;Chat</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Sludring</translation>
     </message>
     <message>
         <location filename="../clientdlgbase.ui" line="531"/>
         <source>C&amp;onnect</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Koble til</translation>
     </message>
     <message>
         <location filename="../clientdlgbase.ui" line="557"/>
         <source>MUTED (Other people won&apos;t hear you)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Forstummet (andre hører deg ikke)</translation>
```

Comment:
I think Forstummet should also be all caps to reassemble the English version

Reply:
I don't really want to resemble the English version.
The nb_NO one is consistent, and the user needs never know the English one does anything differently.

https://www.compart.com/en/unicode/search?q=speaker#characters or something like
a crossed out person talking, a crossed out speech bubble, etc. would work. I think that is what others do.
![bilde](https://user-images.githubusercontent.com/13802408/191053199-15d2cac2-2c42-406f-8c7b-16184432d644.png)